<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Honey" failures="2" tests="5" name="testRunners.RunCucumberTest" time="187.710" errors="1" timestamp="2025-09-09T06:30:03 IST" skipped="0">
  <testcase classname="testRunners.RunCucumberTest" name="runScenario" time="20.171">
    <failure message="Registration failed expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Registration failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.RegistrationSteps.i_should_be_redirected_to_home_page_without_error(RegistrationSteps.java:248)
at ✽.I should be redirected to the home page without error(file:///C:/Users/dell/Desktop/project/ThomasCookAutomation/ThomasCookAutomation/src/test/resources/features/registration.feature:35)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunners.RunCucumberTest" name="runScenario" time="18.028"/>
  <testcase classname="testRunners.RunCucumberTest" name="runScenario" time="22.417">
    <failure message="Unexpected error message! expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unexpected error message! expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.RegistrationSteps.i_should_see_error_messages_for_invalid_inputs(RegistrationSteps.java:242)
at ✽.I should see error messages for invalid inputs(file:///C:/Users/dell/Desktop/project/ThomasCookAutomation/ThomasCookAutomation/src/test/resources/features/registration.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunners.RunCucumberTest" name="runScenario" time="88.806"/>
  <testcase classname="testRunners.RunCucumberTest" name="runScenario" time="38.288">
    <error message="Expected condition failed: waiting for element to be clickable: By.id: sendOTP (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.23.0&#039;, revision: &#039;4df0a231af&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.12&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 139.0.7258.157, chrome: {chromedriverVersion: 139.0.7258.154 (9e0d6b2b47f..., userDataDir: C:\Users\dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58469}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58469/devtoo..., se:cdpVersion: 139.0.7258.157, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 1064a9116120ae9ccad1e62b029988f2" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.id: sendOTP (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: '4.23.0', revision: '4df0a231af'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 139.0.7258.157, chrome: {chromedriverVersion: 139.0.7258.154 (9e0d6b2b47f..., userDataDir: C:\Users\dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58469}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58469/devtoo..., se:cdpVersion: 139.0.7258.157, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 1064a9116120ae9ccad1e62b029988f2
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at pages.LoginPage.login(LoginPage.java:65)
at stepDefinitions.LoginSteps.i_enter_login_details_and_send_otp(LoginSteps.java:28)
at ✽.I enter login details "invalidUser9" and send OTP(file:///C:/Users/dell/Desktop/project/ThomasCookAutomation/ThomasCookAutomation/src/test/resources/features/login.feature:6)
]]>
    </error>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRunners.RunCucumberTest -->
